debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -959,6 +969,87 @@ protected Object convertBinary(Column column, Field fieldDefn, Object data) {
         return super.convertBinary(column, fieldDefn, (data instanceof PGobject) ? ((PGobject) data).getValue() : data);
     }
 
+    protected Object convertBinary(Column column, Field fieldDefn, Object data, BinaryHandlingMode mode) {
```

Comment:
So this implementation looks solid, but it all should go up to `JdbcValueConverters`, adjusting the existing `convertBinary()` method. That way, not only bytea, but also other binary types benefit from that option, including unknown column types in case of Postgres (CC @hashhar).

Reply:
DONE